subprojects/docs/src/samples/providers/propertyAndProvider/kotlin/build.gradle.kts

```diff
@@ -1,19 +1,19 @@
 open class Greeting : DefaultTask() {
     // Configurable by the user
-    @get:Input
-    val message: Property<String> = project.objects.property()
+    @Input
+    val greeting: Property<String> = project.objects.property()
```

Comment:
❓ Why don't you need this annotation on the `@get:` anymore??
Are you sure this works? @bamboo @eskatos?